CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] - UNRELEASED
```

Comment:
Can we add a date now that we're releasing?

Reply:
I usually update the release date in a separate release PR, just so I don't have to go change it again if something gets delayed.